doc/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
   unset(CMAKE_DOXYGEN_TUTORIAL_CONTRIB_ROOT)
   unset(CMAKE_DOXYGEN_TUTORIAL_JS_ROOT)
 
+  set(OCV_MATHJAX_RELPATH "http://cdn.mathjax.org/mathjax/latest" CACHE STRING "URI to a MathJax installation")
```

Comment:
"URI to a MathJax installation"
URI -> URL ?

Reply:
Because it can also point to a local path (i.e., in Debian we make it point to /usr/share/javascript/mathjax/), therefore 'URL' would be too limited.
What do you think?